suites/configurations/inputs-softlayer.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
see the modified inputs here
https://github.com/cloudify-cosmo/cloudify-manager-blueprints/blob/master/softlayer/inputs.yaml.template
, rename this file to inputs-softlayer.yaml and modify this file to be more YAML like